test/e2e/fixtures.go

```diff
@@ -168,6 +168,44 @@ func setupTestWithIPFIXCollector(tb testing.TB) (*TestData, error, bool) {
 	return data, nil, isIPv6
 }
 
+func setupTestWithFlowAggregator(tb testing.TB) (*TestData, error) {
+	data := &TestData{}
```

Comment:
Why do we need a brand new `TestData` instance instead of using the existing one, which already has an initialized K8s client?

The same question applies to `setupTestWithIPFIXCollector`

Reply:
Thanks, addressed.